salt/grains/core.py

```diff
@@ -1513,6 +1513,7 @@ def id_():
     'Funtoo': 'Gentoo',
     'AIX': 'AIX',
     'TurnKey': 'Debian',
+    'AstraLinuxCE': 'Debian',
```

Comment:
I couldn't really understand if the list has any order, so I hope adding it at the end will be fine. Let me know otherwise.

Reply:
Doesn't look like there's any order. I would have alphabetized it :joy: but I definitely don't expect you to.